src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.RenameTypeEditor.cs

```diff
@@ -29,8 +29,7 @@ public override async Task<Solution> GetModifiedSolutionAsync()
                 // if no such conflicts exist, proceed with RenameSymbolAsync.
                 var solution = SemanticDocument.Document.Project.Solution;
                 var symbol = State.SemanticDocument.SemanticModel.GetDeclaredSymbol(State.TypeNode, CancellationToken);
-                var documentOptions = await SemanticDocument.Document.GetOptionsAsync(CancellationToken).ConfigureAwait(false);
-                return await Renamer.RenameSymbolAsync(solution, symbol, FileName, documentOptions, CancellationToken).ConfigureAwait(false);
+                return await Renamer.RenameSymbolAsync(solution, symbol, new SymbolRenameOptions(), FileName, CancellationToken).ConfigureAwait(false);
```

Comment:
i'm curious why we're not getting options created from teh document. is that because there are no config-oriented options for rename?  and only UI oriented ones?

Reply:
Rename options are not in the snapshot. They all come from inline rename dashboard UI.

Reply:
BTW, this is actually the ultimate goal. Options that can't be configured via .editorconfig should not be read from document/solution.